src/main/java/uk/gov/companieshouse/api/accounts/model/entity/AccountDataEntity.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.companieshouse.api.accounts.model.entity;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+import javax.validation.constraints.NotNull;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class AccountDataEntity {
+
+    @Field("id")
+    private String id;
```

Comment:
this class is represents an inner 'data' subdocument within a mongodb document so doesn't need an id field, the mongo id is in the wrapper AccountEntity class